BUILD.md

```diff
@@ -147,6 +147,30 @@ The `docs` target uses the `ghcr.io/kanisterio/docker-sphinx` public image to
 generate the HTML documents and store them in your local `/docs/_build/html`
 folder.
 
+## New Documentation
+
+We have started experimenting, and will soon fully transition, to using [VitePress](https://vitepress.dev/) to generate Kanister documentation.
```

Comment:
[nit] Could we add line breaks at ~80 chars? We may not have followed it strictly but would like to going forward :)

Reply:
Done.